reana_workflow_engine_snakemake/executor.py

```diff
@@ -10,288 +10,95 @@
 
 import os
 import logging
-import asyncio
-from collections import namedtuple
-from typing import Callable
+from pathlib import Path
+
+from snakemake.api import SnakemakeApi
+from snakemake.settings.types import (
```

Comment:
As discussed IRL, the files could be renamed as follows:

- `executor.py` -> `runner.py`
- `reana_executor.py` -> `executor.py`